client/thirdparty.go

```diff
@@ -0,0 +1,54 @@
+package client
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	k8sApiUnversioned "k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/client/restclient"
+)
+
+// ThirdPartyClient can be used to access third party resources
+type ThirdPartyClient struct {
+	*restclient.RESTClient
+	baseURL string
+}
+
+func (c *ThirdPartyClient) Workflows(namespace string) WorkflowInterface {
+	return newWorkflows(c, namespace)
+}
+
+// NewThirdparty creates a new ThirdPartyClient
+func NewThirdparty(gv *k8sApiUnversioned.GroupVersion, c *restclient.Config) (*ThirdPartyClient, error) {
```

Comment:
Dit is dus met die naming conventie niet het idee. Je doe dat client.NewThirdParty(..).
Ik zal wel de casing nog ff aanpassen.